Tests/RxCocoaTests/DelegateProxyTest+UIKit.swift

```diff
@@ -41,6 +41,14 @@ extension DelegateProxyTest {
     }
 }
 
+// MARK: UITabBarController
```

Comment:
This is not a UITabBarController ;), but I can change this after the merge. Cool.